helm/charts/next-template/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+appVersion: "1.0"
+description: Generic Helm Chart for deploying a single tier application
+name: single-tier-application
+version: 0.0.1
+maintainers:
+  - name: Digital Technology Solutions, Site Reliability Engineering
+    email: gocmts@gmail.com
+    url: https://confluence.dts-stn.com/display/OPS/Site+Reliability+Engineering
```

Comment:
Change, add or remove the maintainer to our group. We will be responsible for the template in this app so we don't want people accidentally going straight to SRE for help